Build.props

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <CredentialProviderVersion>0.1.8</CredentialProviderVersion>
```

Comment:
Is this the only place we bump the version?

Reply:
Should be